views/src/app/Auth/components/comments/comments.js

```diff
@@ -26,11 +26,10 @@ class CommentsContainer extends Component {
     postComment = async () => {
         let body = this.commentForm.value;
         try {
-            let res = await axios.post('http://localhost:3001/api/comments/', { body, postItem: this.props.postItem },   
-            { headers: { 'Authorization': `Bearer ${localStorage.token}` } });
+            let res = await axios.post('http://localhost:3001/api/comments/', { body, postItem: this.props.postItem },
```

Comment:
again, you don't want to specify the post and host here to enable this to work in production when you deploy to heroku
```suggestion
            let res = await axios.post('/api/comments/, { body, postItem: this.props.postItem },
```